src/main/java/entities/Profile.java

```diff
@@ -13,82 +16,144 @@ public class Profile implements Serializable {
+    /**
+     * Get all recipes that are reviewed by the user in his/her Profile.
+     */
+    public ArrayList<Review> getReviewed() {
+        return reviewed;
     }
 
+    /**
+     * Get all recipes that are created by the user in his/her Profile.
+     */
+    public ArrayList<Recipe> getCreated() {
+        return created;
+    }
+
+
+    /**
+     * Set the age of the user in his/her Profile.
+     * @param age The input age of the user
+     */
     public void setAge(int age) { this.age = age; }
 
+    /**
+     * Set the height of the user in his/her Profile.
+     * @param height The input height of the user
+     */
     public void setHeight(float height) { this.height = height; }
 
+    /**
+     * Set the weight of the user in his/her Profile.
+     * @param weight The input weight of the user
+     */
     public void setWeight(float weight) { this.weight = weight; }
 
+    /**
+     * Set the gender of the user in his/her Profile.
+     * @param gender The input gender of the user
+     */
+    public void setGender(String gender) {
+        this.gender = gender;
+    }
+
+    /**
+     * Set the reviewed of the user in his/her Profile.
+     * @param reviewed An arraylist of profile which contains all the recipes reviewed by the user.
+     */
+    public void setReviewed(ArrayList<Review> reviewed) {
+        this.reviewed = reviewed;
+    }
+
+    /**
+     * Set the reviewed of the user in his/her Profile.
+     * @param created An arraylist of profile which contains all the recipes created by the user.
+     */
+    public void setCreated(ArrayList<Recipe> created) {
+        this.created = created;
+    }
+
+
+    /**
+     * A method that returns all information of the reviewed recipes of the user in this profile.
+     * @return Return a string which contains all the recipes reviewed by the user.
+     */
     public String soutReviewed() {
-        String result = new String("");
+        StringBuilder result = new StringBuilder();
         for (Review r : reviewed){
-            result = result + r.toString() + "\n";
+            result.append(r.toString()).append("\n");
         }
-        return result;
+        return result.toString();
     }
 
+    /**
+     * A method that returns all information of the created recipes of the user in this profile.
+     * @return Return a string which contains all the recipes created by the user.
+     */
     public String soutCreated() {
-        String result = new String("");
+        StringBuilder result = new StringBuilder();
         for (Recipe r : created){
-            result = result + r.toString() + "\n";
+            result.append(r.toString());
         }
-        return result;
+        return result.toString();
     }
 
+    /**
+     * A method that returns all information of the user in this profile.
+     * @return Return a string that contains the age, height, weight, and gender of the user in this profile.
+     */
     public String soutInfo(){
```

Comment:
what does sout mean/stand for?

Reply:
to print out all information of the user in the profile.